conf/reports/Event.Data_Breach.Relatorio.html

```diff
@@ -15,74 +15,97 @@ <h1 style="text-align: center"> Relat&oacute;rio de Viola&ccedil;&atilde;o de Da
-            {% endfor %}
-            {{ "</table>" }}
-
-    {% else %}
-    <h2> Nenhum Titular Impactado </h2>
+        {{ "
+        <tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>
+            <td style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>%s</td>
+        </tr>
+        " | format (mainkey['Person_Natural_Full_Name'] | default('n/a')) }}
+        {% endfor %}
+        {{ "
+    </table>
+    " }}
     {% endif %}
 
     {% if impacted_data_sources[0] is defined %}
-    <h2> Lista de {{ impacted_data_sources | length }} Fontes de Dados </h2>
-    {{ "<table style='margin: 5px'><tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'><th style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>Nome</th></tr>" }}
-
-    {% for mainkey in impacted_data_sources %}
-    {{  "<tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'><td style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>%s</td></tr>" | format (mainkey['Object_Data_Source_Name'] | default('n/a') )}}
+    <hr/>
+    <h2> Lista de Fontes de Dados </h2>
+    {{ "
+    <table style='margin: 5px'>
+        <tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>
+            <th style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>Nome</th>
+        </tr>
+        " }}
+
+        {% for mainkey in impacted_data_sources %}
+        {{ "
+        <tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>
+            <td style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>%s</td>
+        </tr>
+        " | format (mainkey['Object_Data_Source_Name'] | default('n/a') )}}
         {% endfor %}
-        {{ "</table>" }}
-
-    {% else %}
-    <h2> Nenhuma Fonte de Dados Impactada </h2>
+        {{ "
+    </table>
+    " }}
     {% endif %}
 
     {% if impacted_servers[0] is defined %}
-    <h2> Lista de {{ impacted_servers | length }} M&oacute;dulos </h2>
-
-    {% for mainkey in impacted_servers %}
-    <h3>{{ mainkey.Metadata_Type |replace('.',' ')|replace('_',' ') }}</h3>
-    {{ "<table style='margin: 5px'><tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'><th style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>Nome</th><th style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>Valor</th></tr>" }}
-    {% for key, value in mainkey.items() %}
-    {{  "<tr style='border: 1px solid #dddddd;text-align: left;padding: 8px;'><td style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>%s</td><td style='border: 1px solid #dddddd;text-align: left;padding: 8px;'>%s</td></tr>" | format (key , value )}}
+    <hr/>
+    <h2> Lista de M&oacute;dulos </h2>
+    {{ "

```

Comment:
This looks wrong; why {{ " ??

Reply:
that part ... regarding the data_sources I didn't touch ... in fact ... I used that part as a template to build the other 2 HTML Table ... like you told me so